res/css/views/right_panel/_UserInfo.scss

```diff
@@ -76,6 +81,7 @@ limitations under the License.
     .mx_UserInfo_avatar > div {
         max-width: 30vh;
         margin: 0 auto;
+        transition: 1s;
```

Comment:
Maybe make this faster? I agree with @ara4n that it seems a bit slow (assuming this is the avatar size transition).